projects/gateway-utilities/ignition/script-python/General/TranslationManagementPage/code.py

```diff
@@ -0,0 +1,37 @@
+"""
+General.Translation Management Page
+DESCRIPTION: This page is used to manage translations for a project
+"""
+def import_template(view, event):
+	"""
+	DESCRIPTION: converts a template file from csv into json, and displays it in a preview table
+	PARAMETERS: view (object): the form that is importing the file, event(object): the file to upload
+	RETURNS: null
+	"""
+	file_string = event.file.getString()
+	view.view.custom.data = []
+	for line in file_string.split("\n")[1:]:
+		line = line.split(",")
+		view.view.custom.data.append({
+			"English": line[0],
+			"Spanish": line[1],
+		})
+	
+	
+	view.view.custom.uploaded = True
```

Comment:
Uploading a csv isn't being seen on the table. Some issue with the "append" command or with the bindings not reflecting the changes.

Also, I don't see anywhere where this script is adding to the global translation tabls. Use "system.util.modifyTranslation" to add them.